guide/creating-your-bot/README.md

```diff
@@ -66,6 +66,10 @@ client.on('interactionCreate', interaction => {
 });
 ```
 
+Next, add a Slash Command.
+
+Follow [this page](https://discordjs.guide/interactions/registering-slash-commands.html) to register Slash Command on your Discord server.
```

Comment:
This URL must be a relative path. ([CONTRIBUTING.md#images-and-links](https://github.com/discordjs/guide/blob/main/CONTRIBUTING.md#images-and-links))